nss-cohort-36/keahua-arboretum-net-surfers
# Jl feed animals

# Description

User can now select option 3 and will display animals to be fed as long as a biome and an animal exists

## Type of change
feature addition

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jl-feed-animals`
4. from terminal run python index.py
select option 3 to test logic of whether biome exists
create biome and then select option 3 again to test logic of whether an animal exists
create animal and then option 3 will allow you to feed any animal


# Checklist:
